python/delta/testing/utils.pyi

```diff
@@ -0,0 +1,7 @@
+import unittest
```

Comment:
Depending on preferences test annotations could be dropped, as long as py.typed indicates partial annotations.